pinot-core/src/main/java/org/apache/pinot/core/transport/ServerInstance.java

```diff
@@ -91,6 +102,7 @@ public ServerInstance(InstanceConfig instanceConfig) {
     _queryServicePort = INVALID_PORT;
     _queryMailboxPort = INVALID_PORT;
     _adminEndpoint = null;
+    _group = DEFAULT_SERVER_REPLICA_GROUP_OF_BROKER_VIEW;
```

Comment:
From a semantic point of view a ServerInstance does not have any replica-group concept. Replica-groups are defined at a table level and a given server may belong to different replica-groups for different tables.

Can you think about what this value represents exactly? We can name it accordingly

Reply:
https://docs.pinot.apache.org/operators/operating-pinot/instance-assignment#pool-based-instance-assignment

maybe _pool?